DmitryBarskov/rails-template
# Add Docker template

## Summary
Resolves #22.

Added Docker template which adds Dockerfile and docker-compose depending on the installed database: for postgresql or sqlite.

## Test plan
1. generate a new app with `rails new appname --minimal`
2. apply `docker/base` template by
`bin/rails app:template LOCATION=templates/docker/base.rb`
3. run the server by `docker-compose up`, you should see default welcome
